### PR TITLE
Added class resources reader, and two files with resources

### PR DESCRIPTION
**Pull request by JugerJuger**

### CODE REVIEW

**Review by MelnykNikita — APPROVED** [review]

**Review by MaxKovalets — APPROVED** [review]

File: src/test/java/rv/atqc/gttesting/res/test/RHelperTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	
+	@BeforeClass
+	public void before() {
+		properties = new Properties();
+		InputStream input = null;
+		try {
+			input = R.class.getClassLoader().getResourceAsStream("user-information-test-data.properties");
+			properties.load(input);
+		} catch (IOException ex) {
+			ex.printStackTrace();
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/res/test/RHelperTest.java]: should we stop execution if exception is thrown?

File: src/test/java/rv/atqc/gttesting/res/R.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
+package rv.atqc.gttesting.res;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class R {
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/res/R.java]: please provide meaningful name

File: src/test/java/rv/atqc/gttesting/res/test/RHelperTest.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import rv.atqc.gttesting.res.R;
+
+public class RHelperTest {
```
Comment (dmatsev) [inline src/test/java/rv/atqc/gttesting/res/test/RHelperTest.java]: please provide meaningful name

**Review by dmatsev — APPROVED** [review]
with comments

**Review by CaCO3ua — APPROVED** [review]

**Review by TyshchenkoRoma — APPROVED** [review]

**Pull request merged**